_episodes/05-starting-jekyll.md

```diff
@@ -21,4 +21,57 @@ keypoints:
 - use it again in another page
 - change the value of the global parameter and study the effect
 
+
+> ## Creating your own variable
+>
+> Let's practice making and using variables.
+> Think of a variable you may want to use in your `index.md` page.
+> Some possible options include: your name, the institution you work at, the PI of your lab group.
+> Create the variable and reference it in your `index.md` and save/commit that to your repository.
```

Comment:
If this exercise is meant to show variables in the same document, the solution could show the entire content:
```
---
pi: "Ada Lovelace"
---

This project is lead by {{ page.pi }}.
```

Reply:
Good idea, resolved in recent commit